### PR TITLE
III-5912 Limited name/title to 90 chars

### PR DESCRIPTION
**Pull request by grubolsch**
DO NOT MERGE YET - This also needs work on the frontend or users are going to get confusing error messages.

### Changed

- III-5912 Limited name/title to 90 chars

Related to pr: https://github.com/cultuurnet/udb3-backend/pull/1625

---

Ticket: https://jira.uitdatabank.be/browse/III-5912

### CODE REVIEW

File: projects/uitdatabank/models/common-title.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "title": "copyrightHolder",
```
Comment (LucWollants) [inline projects/uitdatabank/models/common-title.json:2]: This file is called `common-title` but here it is called `copyrightHolder`. This is a new file but nowhere used?

File: projects/uitdatabank/models/common-name-localized.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "title": "name.localized",
   "description": "A human-readable name localized in a single value.",
   "minLength": 1,
+  "maxLength": 90,
```
Comment (LucWollants) [inline projects/uitdatabank/models/common-name-localized.json:6]: Which schemas use this name? Can it cause other fields (accept the name of a place, event and organizer) to have an unwanted limit?

File: projects/uitdatabank/models/common-title.json (hunk 1 above)
Comment (grubolsch) [reply projects/uitdatabank/models/common-title.json]: You are right, it does not do anything. it was a mistake to commit this file.
I have removed it.

File: projects/uitdatabank/models/common-name-localized.json (hunk 2 above)
Comment (grubolsch) [reply projects/uitdatabank/models/common-name-localized.json:6]: I checked and no other usage was detected

**Review by LucWollants — APPROVED** [review]

**Review by JonasVHG — APPROVED** [review]

**Pull request merged**